test/Chainweb/Test/Pact/RemotePactTest.hs

```diff
@@ -255,7 +255,7 @@ testBatch = do
     pure $ SubmitBatch (pure c)
   where
     pm :: CM.PublicMeta
-    pm = CM.PublicMeta (CM.ChainId "0") "sender00" 100 0.1
+    pm = CM.PublicMeta (CM.ChainId "0") "sender00" 100 0.1 1000000 0
```

Comment:
Use a smaller constant in the tests? A few seconds should be enough. In any case it should be smaller than a possibly enforced upper limit. (As long as the test isn't to test the behavior for larger values.)

Reply:
We don't have the functionality in there yet, so I chose an arbitrary value. Changing it would be completely arbitrary at this point.